tests/cupyx_tests/fallback_mode_tests/test_fallback.py

```diff
@@ -31,8 +32,9 @@ def test_func(self, *args, **kwargs):
 
                 if isinstance(numpy_result, numpy.ndarray):
                     # if numpy returns ndarray, cupy must return ndarray
-                    assert isinstance(fallback_result, cupy.ndarray)
-                    testing.assert_array_equal(numpy_result, fallback_result)
+                    assert isinstance(fallback_result, utils.ndarray)
```

Comment:
```suggestion
                    assert isinstance(fallback_result, numpy.ndarray)
```

Reply:
Shouldn't tests be on safe side? I've created `test_instancecheck_ndarray()` for this purpose.

Reply:
I see. How about checking both of them?

Reply:
Ah, it is already tested in `test_instancecheck_ndarray()` as you commented. Thanks.